distdir_deps.bzl

```diff
@@ -488,39 +489,39 @@ DIST_DEPS = {
         "used_in": [],
     },
     "openjdk_macos_x86_64_vanilla": {
-        "archive": "zulu20.30.11-ca-jdk20.0.1-macosx_x64.tar.gz",
-        "sha256": "befee9db92345d5146945061b721d3a6c6e182471c1536f87dbadfd5aab0e241",
-        "strip_prefix": "zulu20.30.11-ca-jdk20.0.1-macosx_x64",
+        "archive": "zulu21.28.85-ca-jdk21.0.0-macosx_x64.tar.gz",
+        "sha256": "9639b87db586d0c89f7a9892ae47f421e442c64b97baebdff31788fbe23265bd",
+        "strip_prefix": "zulu21.28.85-ca-jdk21.0.0-macosx_x64",
         "urls": [
-            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-macosx_x64.tar.gz",
-            "https://cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-macosx_x64.tar.gz",
+            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-macosx_x64.tar.gz",
+            "https://cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-macosx_x64.tar.gz",
         ],
         "used_in": [
         ],
     },
     "openjdk_macos_aarch64_vanilla": {
-        "archive": "zulu20.30.11-ca-jdk20.0.1-macosx_aarch64.tar.gz",
-        "sha256": "01e59f0160d051524bb16d865652d25d00a85390581737a8f35f89057c80892d",
-        "strip_prefix": "zulu20.30.11-ca-jdk20.0.1-macosx_aarch64",
+        "archive": "zulu21.28.85-ca-jdk21.0.0-macosx_aarch64.tar.gz",
+        "sha256": "2a7a99a3ea263dbd8d32a67d1e6e363ba8b25c645c826f5e167a02bbafaff1fa",
+        "strip_prefix": "zulu21.28.85-ca-jdk21.0.0-macosx_aarch64",
         "urls": [
-            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-macosx_aarch64.tar.gz",
-            "https://cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-macosx_aarch64.tar.gz",
+            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-macosx_aarch64.tar.gz",
+            "https://cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-macosx_aarch64.tar.gz",
         ],
         "used_in": [
         ],
     },
     "openjdk_win_vanilla": {
-        "archive": "zulu20.30.11-ca-jdk20.0.1-win_x64.zip",
-        "sha256": "8a97ee11da578292f7c9e772f3edd3f083fa4f34f47a98e3abefb625ab2225ba",
-        "strip_prefix": "zulu20.30.11-ca-jdk20.0.1-win_x64",
+        "archive": "zulu21.28.85-ca-jdk21.0.0-win_x64.zip",
+        "sha256": "e9959d500a0d9a7694ac243baf657761479da132f0f94720cbffd092150bd802",
+        "strip_prefix": "zulu21.28.85-ca-jdk21.0.0-win_x64",
         "urls": [
-            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-win_x64.zip",
-            "https://cdn.azul.com/zulu/bin/zulu20.30.11-ca-jdk20.0.1-win_x64.zip",
+            "https://mirror.bazel.build/cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-win_x64.zip",
+            "https://cdn.azul.com/zulu/bin/zulu21.28.85-ca-jdk21.0.0-win_x64.zip",
         ],
         "used_in": [
         ],
     },
-    # JDK20 unavailable so use JDK19 instead for win aarch64.
+    # JDK21 unavailable so use JDK19 instead for win aarch64.
```

Comment:
We should keep in mind that this means that using JDK 21 features such as virtual threads will fail at runtime with Win aarch64 builds of Bazel.

How about using Microsoft's build of the OpenJDK for this platform? https://learn.microsoft.com/en-us/java/openjdk/download#openjdk-21

CC @meteorcloudy

Reply:
Yup! @coeuvre and I were considering this as well, as for linux s390x and ppc64le, we don't release any Bazel binaries for those so it should be fine to ignore those.

@meteorcloudy @meisterT any objections against using Microsoft's OpenJDK build?

Reply:
No

Reply:
Done.